ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/KsqlRestApplication.java

```diff
@@ -406,8 +430,53 @@ private void displayWelcomeMessage() {
       }
     }
   }
-}
 
-/*
-        TODO: Find a good, forwards-compatible use for the root resource
- */
+  public void unZipIt(File zipFile, String outputFolder){
```

Comment:
These three methods below can be private and static. TBH, feels like they should be in a util class...